modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -93,6 +93,10 @@ package object dao extends MoreTupleOps {
   implicit val OffsetQMeta: Meta[Offset.Q] =
     AngleMetaAsSignedArcseconds.xmap(Offset.Q(_), _.toAngle)
 
+  // Wavelength maps to an integer in angstroms
+  implicit val WavelengthMeta: Meta[Wavelength] =
+    Meta[Int].xmap(Wavelength.unsafeFromAngstroms, _.toAngstroms)
+
```

Comment:
So, I think we may want to leave this non-implicit and have specialized wrappers for `GmosWavelength` or something that map to an exact decimal representation in nanometers (as we did with offsets, which are stored as decimal arcseconds) if that's what a human wants to see. I think it's important to be able to look at a database table and see values that make intuitive sense. We need to add comments to the columns in question, now that I think about it.

But if it's the case that every wavelength in the science model (as far as we know) makes sense in angstroms then this mapping seems fine to me.